manual/core/query_timestamps/README.md

```diff
@@ -0,0 +1,137 @@
+## Query timestamps
+
+In Cassandra, each mutation has a microsecond-precision timestamp, which is used to order operations
+relative to each other.
+
+There are various ways to assign it:
+
+### CQL `USING TIMESTAMP`
+
+You can explicitly provide the timestamp in your CQL query:
+
+```java
+session.execute("INSERT INTO my_table(c1, c2) values (1, 1) " +
+    "USING TIMESTAMP 1432815430948040");
+```
+
+### Timestamp generator
+
+The driver has a timestamp generator that gets invoked for every outgoing request; it either assigns
+a client-side timestamp to the request, or indicates that the server should assign it. 
+
+The timestamp generator is defined in the [configuration](../configuration/).
+
+#### `AtomicTimestampGenerator`
```

Comment:
In our generated docs, sections headings using backticks look a bit awkward, it might be better to not use them (although opinion on that is subjective).

Reply:
I agree, I used plain text in the retries page, I'll do the  same here.